src/pages/changelog/2025-10-8.mdx

```diff
@@ -0,0 +1,11 @@
+---
+date: 2025-10-08
+title: Introducing sequential signing
```

Comment:
It reads a little weird when ALL titles start with "Introducing" :D 
Any other suggestions?

Reply:
@nmoskaleva Yes, by accident, not by design :D Mostly me and Lasse not being super creative.

Reply:
Changed to something a little different